pkg/coordinator/wallet.go

```diff
@@ -0,0 +1,25 @@
+package coordinator
+
+import (
+	"fmt"
+
+	"github.com/keep-network/keep-core/pkg/internal/hexutils"
+)
+
+// NewWalletPublicKeyHash decodes a wallet public key hash from a string.
+func NewWalletPublicKeyHash(str string) ([20]byte, error) {
```

Comment:
b63d4a94366ce9fae857650ebb6a0d9c84cd2378

Reply:
Yes, I wanted to do it exactly as you described, but due to the usage of `hexutils` package which sits in `pkg/internal` and wasn't reachable from `cmd` package I wanted to avoid a lot of changes in this PR.

I moved the internal utils package one level up in d45ade09db439484a1c6f713c889056ef4207c64 and moved the wallet public key hash decoding function in 083e8cb3e484ce330881ed843cc66df1f2dfb172.

Should be good now.